LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -37,80 +69,415 @@
+		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;CollateralizedSecuredLoan"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;isServicedBy"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;Servicer"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;assumes"/>
+				<owl:onClass rdf:resource="&fibo-loan-typ-mtg;Mortgage"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-gty;hasGuarantor"/>
+				<owl:onClass rdf:resource="&fibo-loan-typ-mtg;MortgageIndemnityGuarantor"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;isIdentifiedBy"/>
+				<owl:onClass rdf:resource="&fibo-loan-typ-mtg;UniversalLoanIdentifier"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasClosingDate"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-dt-fd;Date"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasOriginatingServiceProvider"/>
+				<owl:someValuesFrom>
+					<owl:Class>
+						<owl:intersectionOf rdf:parseType="Collection">
+							<rdf:Description rdf:about="&fibo-fbc-pas-fpas;FinancialServiceProvider">
+							</rdf:Description>
+							<owl:Restriction>
+								<owl:onProperty rdf:resource="&lcc-lr;isIdentifiedBy"/>
+								<owl:someValuesFrom rdf:resource="&fibo-loan-typ-mtg;NMLSR-ID"/>
+							</owl:Restriction>
+						</owl:intersectionOf>
+					</owl:Class>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-mtg;hasOriginatorPerson"/>
+				<owl:someValuesFrom>
+					<owl:Class>
+						<owl:intersectionOf rdf:parseType="Collection">
+							<rdf:Description rdf:about="&fibo-be-le-lp;LegallyCompetentNaturalPerson">
+							</rdf:Description>
+							<owl:Restriction>
+								<owl:onProperty rdf:resource="&lcc-lr;has"/>
+								<owl:someValuesFrom rdf:resource="&fibo-loan-typ-mtg;NMLSR-ID"/>
+							</owl:Restriction>
+						</owl:intersectionOf>
+					</owl:Class>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>mortgage</rdfs:label>
+		<owl:equivalentClass>
```

Comment:
this says a Mortgage is any Loan which has a SecurityAgreement with RealEstate as collateral. Not true - all sorts of loans can be secured on property.

Reply:
Agreed.  Needs SME review.